manifests/istio-control/istio-discovery/templates/service.yaml

```diff
@@ -17,6 +17,14 @@ spec:
       targetPort: 15017
     - port: 15014
       name: http-monitoring # prometheus stats
+    - name: dns
+      port: 53
+      targetPort: 15053
+      protocol: UDP
```

Comment:
What’s the point of udp when we are trying to do dns over tls

Reply:
There are few points:
- we'll need to do a lot of perf testing, and it's good to characterize the overhead and price of TLS. 
- networks with ipsec or equivalent don't need DNS-over-TLS - but we still want to be able to delegate .global to istiod. 
- troubleshooting and testing are much easier